### PR TITLE
Add 2.4.5

### PR DESCRIPTION
**Pull request by lukasz-golebiewski**

### CODE REVIEW

**Comment by jimmidyson** [comment]
ok to test

File: pom.xml [hunk 1]
```diff
@@ -54,7 +54,7 @@
 
   <properties>
     <elasticsearch.plugin.name>cloud-kubernetes</elasticsearch.plugin.name>
-    <elasticsearch.version>2.4.4</elasticsearch.version>
+    <elasticsearch.version>2.4.5</elasticsearch.version>
     <kubernetes-client.version>1.3.83</kubernetes-client.version>
     <lucene.version>5.5.2</lucene.version>
```
Comment (lukasz-golebiewski) [inline pom.xml]: Should lucene version here match the one of ES? I think `2.4.5` uses `5.5.4`

Comment (jimmidyson) [inline pom.xml]: Good catch! Yes please.

**Comment by jimmidyson** [comment]
Thanks! [merge]

**Pull request merged**